.pre-commit-config.yaml

```diff
@@ -4,6 +4,7 @@ repos:
   hooks:
   - id: check-yaml
   - id: check-json
+    exclude: ^hugo/layouts/index\.json\.json$
```

Comment:
Should be removed if we forsake the JSON

Reply:
Good spot! Now removed in commit 697bf79b82abe2978e890b8a43f0ebd73329c3f6 :)